qiita_db/analysis.py

```diff
@@ -282,7 +282,7 @@ def shared_with(self):
 
     @property
     def all_associated_filepath_ids(self):
-        """Get all associated filepath_ids EXCEPT job results filepaths
```

Comment:
@squirrelo I don't think this is an issue... If you want to retrieve all the files associated with an analysis, I would say that this makes sense. The 2 objects are really related so improving SQL queries rather than rely on python interactions can be really good.